### PR TITLE
add extra metrics options to monitor

### PR DESCRIPTION
**Pull request by nicklasxxx**

### CODE REVIEW

File: MiniTwit/Program.cs [hunk 1]
```diff
@@ -30,6 +31,28 @@
...
+        "Microsoft.AspNetCore.Routing", 
+        "Microsoft.AspNetCore.Diagnostics"
+        );
+    
+    builder.AddView("http.server.request.duration",
+       
+        new ExplicitBucketHistogramConfiguration
+        {
+            Boundaries = new double[] { 0, 0.005, 0.01, 0.025, 0.05,
+                    0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 }
```
Comment (GertLu) [inline MiniTwit/Program.cs:50]: What do these constants mean?

Comment (nicklasxxx) [reply MiniTwit/Program.cs:50]: Don't know, but they are part of the official documentation: 
https://learn.microsoft.com/en-us/aspnet/core/log-mon/metrics/metrics?view=aspnetcore-8.0

Comment (GertLu) [reply MiniTwit/Program.cs:50]: Okay, I think they are the buckets for the histogram of the request duraiton

**Review by GertLu — APPROVED** [review]
This will still need to evolve (ideally this code shouldn't be in program.cs but somewhere else) but for a start it looks good

**Pull request merged**